core/components/menu/menuitem.js

```diff
@@ -25,121 +24,138 @@ goog.require('Blockly.utils.object');
  *     the item.
  * @param {string=} opt_value Data/model associated with the menu item.
  * @constructor
- * @extends {Blockly.Component}
  */
 Blockly.MenuItem = function(content, opt_value) {
-  Blockly.Component.call(this);
+  /**
+   * Human-readable text of this menu item.
+   * @type {string}
+   * @private
+   */
+  this.content_ = content;
 
-  this.setContentInternal(content);
-  this.setValue(opt_value);
+  /**
+   * Machine-readable value of this menu item.
+   * @type {string|undefined}
+   * @private
+   */
+  this.value_ = opt_value;
 
   /**
+   * Is the menu item clickable, as opposed to greyed-out.
    * @type {boolean}
    * @private
    */
   this.enabled_ = true;
 
   /**
-   * @type {Blockly.MenuItem}
+   * The DOM element for the menu item.
+   * @type {?Element}
+   * @private
+   */
+  this.element_ = null;
+
+  /**
+   * Whether the menu item is rendered right-to-left.
+   * @type {boolean}
+   * @private
+   */
+  this.rightToLeft_ = false;
+
+  /**
+   * ARIA name for this menu.
+   * @type {Blockly.utils.aria.Role}
```

Comment:
Role is not an object type but an enum, so this is actually ``{!Blockly.utils.aria.Role}`` similar to ``{string}``. 

Since it's nullable, you'll have to explicitly state that, ie: ``{?Blockly.utils.aria.Role}``